test/commands/track/determine_concepts_layout_test.rb

```diff
@@ -57,12 +58,14 @@ def test_layout_with_two_levels
             {
               slug: 'basics',
               name: 'Basics',
-              web_url: 'https://test.exercism.io/tracks/ruby/concepts/basics'
+              web_url: 'https://test.exercism.io/tracks/ruby/concepts/basics',
+              tooltip_url: '/tracks/ruby/concepts/basics/tooltip'
```

Comment:
```suggestion
              tooltip_url: 'https://test.exercism.io/tracks/ruby/concepts/basics/tooltip'
```

etc

Reply:
Ya, haven't finished all the changes just yet. Ran out of time last night. Will be working on this more today

Reply:
No probs! I figured you knew :)